Sources/PointFree/Episode/Show.swift

```diff
@@ -794,15 +832,21 @@ private enum EpisodePermission: Equatable {
   }
 }
 
-private let pfVideoClass = CssSelector.class("pf-video")
```

Comment:
I got rid of these video specific styles that we rendered into the head, and instead was able to build everything off of our style guide.